src/models/Image.js

```diff
@@ -15,3 +15,9 @@ export type TaggableImage = Image & {
   addTag: (tag: Tag) => void,
   removeTag: (tag: Tag) => void,
 };
+
+export type SocketioImages = {
```

Comment:
I wouldn't put SocketIO in the name here. Aren't this just presentation images? Why is this not the Presentation Type?